core/jvm/src/main/scala/zio/Signals.scala

```diff
@@ -0,0 +1,15 @@
+package zio
+
+import sun.misc.{Signal, SignalHandler}
+import zio.console._
+
+object Signals {
+  def fiberDump: SignalHandler =
+    Signal.handle(new Signal("USR1"), (sig: Signal) => {
+      for {
+      dumps <- Fiber.dump
+      _ <- URIO.foreach(dumps)(_.prettyPrintM.flatMap(console.putStrLn))
```

Comment:
We may want to consolidate all the dumps into one string and print that to make sure that nothing else being printed concurrently is printed in between the dumps.